frame/dapps-staking/src/lib.rs

```diff
@@ -177,10 +93,8 @@ pub enum SmartContract<AccountId> {
 }
```

Comment:
can you just swap Evm and Wasm.
It will be easier to use on the polkadot.js

Reply:
Sure.

But why will it be easier?

Reply:
polkadot.js gives you dropdown menu where first one is default. 
When you have a lot of interaction with the pallet through polkadot.js it saves some time and nerves :)